### PR TITLE
docs troubleshooting

### PR DESCRIPTION
**Pull request by joshka**
- docs: add troubleshooting to development.md

### CODE REVIEW

**Comment by joshka** [comment]
Inadvertently still had the changes from #146 in this - I'll clean this up once #146 is merged.

**Comment by joshka** [comment]
Rebased on main

**Comment by joshka** [comment]
fixed typo

**Review by taiki-e — APPROVED** [review]
Thanks.

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -24,3 +24,10 @@ See JSON files in `tools/codegen/base` directory for examples of the manifest.
 3\. Add tool name to test matrix in `.github/workflows/ci.yml`.
 
 4\. Add tool name to the table in ["Supported tools" section in `README.md`](https://github.com/taiki-e/install-action#supported-tools).
+
+## Troubleshooting
+
+If one of the CI builds fails due to an bin path or release asset_name, fix the problem in the base
+manifest, and re-run the manifest tool to regenerate the manifest json file. The base manifest
```
Comment (NobodyXu) [inline DEVELOPMENT.md:31]:
```suggestion
manifest, and re-run the manifest tool `tools/manifest.sh` to regenerate the manifest json file. The base manifest
```

**Review by NobodyXu — CHANGES_REQUESTED** [review]

File: DEVELOPMENT.md (hunk 1 above)
Comment (joshka) [reply DEVELOPMENT.md:31]: Done

Comment (NobodyXu) [reply DEVELOPMENT.md:31]: Well it's already merged 😂

**Comment by joshka** [comment]
whoops missed the merge (another PR incoming)

**Pull request merged**